SEED-platform/seed
# swap label and filter positions

#### Any background context you want to provide?
DC users suggested a new user might be inclined to think that "Filter By Label" and "Current Filters" are linked through the "AND OR EXCL".

#### What's this PR do?
Swaps positions of "Filter by Label" and "Current Filters" to prevent confusion.

#### How should this be manually tested?
navigate to inventory beta, confirm "Current Filters" section is above "Filter By Label"

#### What are the relevant tickets?
#3256 

#### Screenshots (if appropriate)
New
![Screen Shot 2022-05-19 at 12 43 34 PM](https://user-images.githubusercontent.com/58446472/169377732-20c706bf-fa9d-4668-a9d1-c5f8f7ea62e6.png)

Original
<img width="682" alt="Screen Shot 2022-05-19 at 12 48 30 PM" src="https://user-images.githubusercontent.com/58446472/169377737-5148b7c7-38cc-478c-aa5a-1b1355fb28c9.png">
